vite.config.js

```diff
@@ -23,5 +25,8 @@ export default defineConfig({
       ),
       '@images': path.resolve(__dirname, 'app/frontend/images')
     }
+  },
+  test: {
+    globals: true
```

Comment:
I think there's an argument for not using these globals, but we can handle that separately if we think it's a good idea. Doing it like this makes sense for migrating from Jest so I'm happy with it in this PR.

Reply:
💯  but to keep this PR smaller and focused, lets keep it as is and do another PR for globals